docs/source/get-started/android/issues.rst

```diff
@@ -1,8 +1,96 @@
-Common issues when developing Android apps
-===============================================
+Common issues when developing Android SIP client
+==================================================
+
+.. contents:: Common issues:
+   :local:
+   :depth: 2
+
+
+Pjsua2 keeps stopping during startup
```

Comment:
Is this section intentionally empty? Or is this supposed to be the parent of all the "failed" items below?

Reply:
oops.. missed that